services/mysql/init-scripts/my.cnf

```diff
@@ -0,0 +1,8 @@
+[mysqld]
+gtid_mode = ON
+enforce-gtid-consistency = ON
+# Row format required for ZongJi
+binlog_format = row
+log_bin=mysql-bin
+server-id=1  
+binlog-do-db=powersync
```

Comment:
I am wondering if we should ad a filter for the tables in the sync rules here?
That could be good for this example use case. Without the filter the binlog can get large very quickly

Reply:
This database only has two tables, which are both used in the sync rules. Is this still relevant? What else would cause the Binlog to increase in size?

Reply:
Oh it is not so much for the sake of the demo, but rather the case that users might just copy these settings for their own project and not know that one should be filtering the binlog like that.
Of course this is also solveable with good documentation from our side.

Reply:
Yeah, I think we should document this well either way. I'd be happy to add a link here to that documentation when it's available.